src/containers/CompanionWindow.js

```diff
@@ -14,13 +17,16 @@ import { CompanionWindow } from '../components/CompanionWindow';
  * @private
  */
 const mapStateToProps = (state, { id, windowId }) => {
+  const window = getWindow(state, { windowId });
   const companionWindow = getCompanionWindow(state, { companionWindowId: id });
+  const showMoveToBottom = !window.height || window.height > 300 || companionWindow.position !== 'right';
```

Comment:
@glefi I believe the desired behavior here is to `disable` the button, not necessarily remove it all-together.  I think this should be a value that would be passed to the appropriate `MiradorMenuButton` component as the `disabled` prop (which would also indicate a potential name/logic change).